apps/cms/pages/models.py

```diff
@@ -15,31 +19,80 @@ class HomePage(Page):
         verbose_name="Header Image",
         help_text="The Image that is shown on top of the page"
     )
-    subtitle = models.CharField(
+
+    subtitle_de = models.CharField(
+        max_length=500, blank=True, verbose_name="Subtitle")
+    subtitle_en = models.CharField(
         max_length=500, blank=True, verbose_name="Subtitle")
-    body = RichTextField(blank=True)
 
-    subpage_types = ['a4_candy_cms_pages.EmptyPage']
+    body_de = RichTextField(blank=True)
```

Comment:
Should we add a `verbose_name` here? Would make it look nicer (and makes the translation easier?)

Reply:
Or do we plan to ever use this somewhere else?